playwright/tests/changes/changes.spec.ts

```diff
@@ -1,5 +1,134 @@
+import { expect } from "@playwright/test";
+
 import { test } from "../../fixtures";
+import { mockManagerProcedures } from "../../utils";
+import { emptyLibraries, simpleCustomType } from "../../mocks";
+
+test.describe("Changes", () => {
+  test.run({ loggedIn: true })(
+    "I cannot see the login screen when logged in",
+    async ({ changesPage }) => {
+      await changesPage.goto();
+      await expect(changesPage.loginButton).not.toBeVisible();
+      await expect(changesPage.notLoggedInTitle).not.toBeVisible();
+      await expect(changesPage.notAuthorizedTitle).not.toBeVisible();
+    },
+  );
+
+  test.run()(
+    "I can see the login screen when logged out",
+    async ({ changesPage }) => {
+      await changesPage.goto();
+      await expect(changesPage.loginButton).toBeVisible();
+      await expect(changesPage.notLoggedInTitle).toBeVisible();
+    },
+  );
+
+  test.run({ loggedIn: true })(
+    "I can see the unauthorized screen when not authorized",
+    async ({ changesPage }) => {
+      await mockManagerProcedures({
+        page: changesPage.page,
+        procedures: [
+          {
+            name: "getState",
+            data: (data) => ({
+              ...data,
+              clientError: {
+                status: 403,
+              },
+            }),
+          },
+        ],
+      });
```

Comment:
I'm not sure we should be mocking the manager's responses. It removes some of the test's thoroughness since we want to test that the manager is working properly. Is it possible to mock the external APIs instead? I think we would want to mock APIs as far away from Slice Machine's code as possible.

Reply:
It's 100% sure that mocking should be used a little as possible since it's e2e. But for standalone tests, opposite as full scenario you want to be able to test on any running Slice Machine some default use case. 

It will break the e2e management but ensure consistency in your front-end. I 100% agree that it would be better to mock the API behind, but I have no clue how to do and if it's possible.  Do you think we can mock the API since Plawright mock what the browser can see?